src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -108,12 +110,15 @@ private void initialize() {
                     container.getStyleClass().add("entry-container");
                     BindingsHelper.includePseudoClassWhen(container, entrySelected, addToggle.selectedProperty());
 
-                    if (viewModel.hasDuplicate(entry)) {
-                        Button duplicateButton = IconTheme.JabRefIcons.DUPLICATE.asButton();
-                        duplicateButton.setTooltip(new Tooltip(Localization.lang("Possible duplicate of existing entry. Click to resolve.")));
-                        duplicateButton.setOnAction(event -> viewModel.resolveDuplicate(entry));
-                        container.getChildren().add(1, duplicateButton);
-                    }
+                    Callable<Boolean> hasDuplicateEntryTask = () -> viewModel.hasDuplicate(entry);
+                    BackgroundTask.wrap(hasDuplicateEntryTask).onSuccess(e -> {
```

Comment:
```suggestion
                    BackgroundTask.wrap(viewModel::hasDuplicate).onSuccess(e -> {
```
or if it's not possible you can inline the callable as folows
```suggestion
                    BackgroundTask.wrap(()->viewModel.hasDuplicate(entry)).onSuccess(e -> {
```